docker-compose.yml

```diff
@@ -0,0 +1,23 @@
+version: '3'
+
+services:
+  db:
+    image: postgres:10-alpine
+    volumes:
+      - ./var/data/pgsql/:/var/lib/postgresql
+    env_file: '.docker/paste/postgres_env'
+  paste:
+    build: ./.docker/paste
+    volumes:
+      - .:/app
+    depends_on:
+      - db
+  nginx:
+    build: ./.docker/nginx
+    ports:
+      - "80:80"
+    volumes:
+      - ./var/log/nginx:/var/log/nginx
```

Comment:
This should probably go in the ignore folder with the configs.

Reply:
i'm not fond of the idea of having to shell into the machine to read logs, but to each their own

Reply:
That's not what I'm advocating for. I'm saying the logs should be in the folder on the host that has the configuration files.

Reply:
To use this comment to reply to the Line 22 comment (I cannot reply to it directly)

what do you mean with `docker exec`? at this point all you're running is `docker-compose` (barring the inefficient post-install scripts.)

Reply:
`docker exec -ti paste_paste_1 sh /root/start.sh`

Does that not have to be run after `docker-compose up`? That's what I'm asking about. Shouldn't `docker-compose up` handle starting paste?

Reply:
yes, normally the `CMD` does this, but the binary isn't built and the db is not online.